api/javascript/es2015-nodejs/README.md

```diff
@@ -0,0 +1,87 @@
+# GitHub API + ES2015 + node.js
```

Comment:
@nathos thanks a lot 😄 

I use these source code to set up GitHub enterprise demos => I will add soon more features and samples (about CI and status)